README.md

```diff
@@ -21,7 +21,7 @@ In order to use the SSO authentication your browser needs to talk to your app on
 
 You only need to do that once (well, maybe again if you upgrade your OS — that _might_ update your `hosts` file).
 
-In order to get your app to listen on port `80` you will need to make sure the following command is running whenever you run your node server:
+In order to get your app to listen on port `80` you will need to make sure the following command is running *before every time* you run your node server:
```

Comment:
Can I suggest we remove this line altogether and just update the default `callbackUrl` to use port 3001 (i.e. `http://local.nypl.org/callback:3001`)?

Reply:
If we are going to just assign the `callbackUrl ` to port 3001, we don't need this README then